src/demo-app/a11y/toolbar/toolbar-a11y.html

```diff
@@ -0,0 +1,26 @@
+<div class="demo-toolbar">
+  <section>
+    <h2>Basic Toolbar with Text  (e.g. Only display app’s name)</h2>
+    <md-toolbar>My App</md-toolbar>
+  </section>
+
+  <section>
+    <h2>Multiple Lines Toolbar</h2>
+    <md-toolbar>Help</md-toolbar>
+    <md-toolbar>Settings</md-toolbar>
+  </section>
+
+  <section>
+    <h2>Toolbar with favorite icon</h2>
+    <md-toolbar>
+      <span>My App</span>
+      <span class="example-spacer"></span>
+      <md-icon aria-label="favorite">favorite</md-icon>
```

Comment:
Is `aria-label` working on `md-icon`? Consider adding a span with cdk-visually-hidden or make it a button/link? 
https://material.angular.io/components/icon/overview